rfc/cart-api-v1.md

```diff
@@ -0,0 +1,84 @@
+# Version 1
+
+| Issue | Status |
+| ----- | ------ |
+| #88   | Draft  |
+
+## Data Models
+
+### `Cart`
+| Property | Type | Description |
+| --- | --- | --- |
+| `id` | UUID-like `str` | Cart ID, auto-generated by the web API |
+| `user_id` | UUID-like `str` | The ID of the user that creates the collection |
+| `cart_name` | str | Cart Name
+| `entity_type` | str | Entity Type |
```

Comment:
Carts should be able to mix items of a different types.

Reply:
According to the comment from @theathorn (https://github.com/DataBiosphere/azul/issues/88#issuecomment-421520215), the cart should be per entity type. Is this requirement still standing?